src/UKHO.SAP.MockAPIService/UKHO.SAP.MockAPIService.csproj

```diff
@@ -23,4 +23,8 @@
     <ProjectReference Include="..\UKHO.ERPFacade.Common\UKHO.ERPFacade.Common.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Data\" />
```

Comment:
We dont need to use physical folder in project structure. Lets use HOME directory of webapp.

Reply:
Done.